src/utils/playGame.jsx

```diff
@@ -131,6 +133,59 @@ const PlayGame = (e) => {
 			console.log("COMPUTER Win");
 		}
 	}
+
+	let gameHistoryId;
+
+	async function initHistory() {
+		try {
+			let res = await axios({
+				method: 'post',
+				url: '/games/play/init',
+				headers: { 
+				  'Authorization': `Bearer ${localStorage.getItem('token')}`
+				},
+				data : {
+					id: 1,
+					playedAt: new Date().toISOString()
+				}
+			})
+			gameHistoryId = res.data.data.id;
+			console.log(res);
+		} catch(err) {
+			console.log(err)
+		}
+	}
+
+	async function updateHistory() {
+		try {
+			let res = await axios({
+				method: 'post',
+				url: '/games/play/com',
+				headers: { 
+				  'Authorization': `Bearer ${localStorage.getItem('token')}`
+				},
+				data : {
+					id: 1,
+					idHistory: gameHistoryId,
+					status: result.toUpperCase(),
+					metaData: {
+						playerChoice: playerChoice.toUpperCase(),
+						comChoice: compChoice.toUpperCase()
+					}
+				}
+			})
+			console.log(res);
+		} catch(err) {
+			console.log(err)
+		}
+	}
+
+	async function createHistory() {
+		await initHistory();
+		await updateHistory();
```

Comment:
mas kalau kaya gini bukannya init dan updatenya jd bareng ya? yg dimau bukannya init itu pas player milih, update pas computer udah milih. cmiiw

Reply:
@hrz8 mas jadinya aku bikin function play gamenya as async function ya..
jadi computer baru ngerandomize suitnya setelah init berhasil
dan updateHistory + updateBadge setelah result game keluar

Reply:
@hrz8 code yang sebelumnya meng-init dan meng-update data setelah semua variable ready mas (playerchoice, computerchoice, result)

Reply:
oh oke mas got it